### ISSUE TITLE
Delete Page bug

### ISSUE THREAD

**Issue by miketweaver** [LANCommander/LANCommander#113]
```
[18:34:39 ERR] Error deleting page!
System.InvalidOperationException: The instance of entity type 'Page' cannot be tracked because another instance with the same key value for {'Id'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.ThrowIdentityConflict(InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.Add(TKey key, InternalEntityEntry entry, Boolean updateDuplicate)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.Add(TKey key, InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.Add(InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.StartTracking(InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.InternalEntityEntry.SetEntityState(EntityState oldState, EntityState newState, Boolean acceptChanges, Boolean modifyProperties)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.InternalEntityEntry.SetEntityState(EntityState entityState, Boolean acceptChanges, Boolean modifyProperties, Nullable`1 forceStateWhenUnknownKey, Nullable`1 fallbackState)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityGraphAttacher.PaintAction(EntityEntryGraphNode`1 node)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityEntryGraphIterator.TraverseGraph[TState](EntityEntryGraphNode`1 node, Func`2 handleNode)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityGraphAttacher.AttachGraph(InternalEntityEntry rootEntry, EntityState targetState, EntityState storeGeneratedWithKeySetTargetState, Boolean forceStateWhenUnknownKey)
   at Microsoft.EntityFrameworkCore.DbContext.SetEntityState(InternalEntityEntry entry, EntityState entityState)
   at Microsoft.EntityFrameworkCore.DbContext.Remove[TEntity](TEntity entity)
   at LANCommander.Server.Data.Repository`1.Delete(T entity) in D:\a\LANCommander\LANCommander\LANCommander.Server\Data\Repository.cs:line 67
   at LANCommander.Server.Services.BaseDatabaseService`1.Delete(T entity) in D:\a\LANCommander\LANCommander\LANCommander.Server\Services\_BaseDatabaseService.cs:line 111
   at LANCommander.Server.UI.Pages.Pages.Edit.Delete() in D:\a\LANCommander\LANCommander\LANCommander.Server\UI\Pages\Pages\Edit.razor:line 233
   ```

![image](https://github.com/user-attachments/assets/c944fd71-e098-4863-8056-5359636193f3)

https://github.com/user-attachments/assets/438559aa-ef14-47c7-96ab-c7391f10bcb1

**Comment by miketweaver** [comment]
There's actually a ton of bugs around this.

https://github.com/user-attachments/assets/cfe12992-3326-4a70-acfc-7457b3ab085d

Issues found:
Can't delete pages sometimes
Can't create pages sometimes
Editing the slug adds a `new` or `n` 
After deleting the page, the pages on the left don't disappear

**Comment by pathartl** [comment]
Reopening, didn't see the other bugs

**Comment by pathartl** [comment]
Code fixes written, will be deployed in 1.0.1
